### ISSUE TITLE
[line_length_disable_closure_parameter_position] Disable closure_parameter_position if line is too long

### ISSUE THREAD

**Issue by hfehrmann** [realm/SwiftLint#2757]
### New Issue Checklist

- [x] Updated SwiftLint to the latest version
- [x] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

Allow to disable the `closure_parameter_position` if line of the closure is already to big. The rule would take a number or be inherited from `line_length`. The idea is to be able to conform to [Google Swift Style Guide for function calls](https://google.github.io/swift/#function-calls). This rule should be opt-in.

#### Should be allowed

```swift
someVeryLongNameObject.someLongFunctionCallWhichBecomeBiggerThanLineLength { 
    (argument1, ..., argumentN) in 
    //method body
} 
```

**Comment by kapitoshka438** [comment]
Any updates on this request? Long closure parameters list really conflict with the `line_length` rule.
